ui/src/components/system/EnvConfig.vue

```diff
@@ -31,10 +76,16 @@ import { AxiosError, AxiosResponse } from 'axios'
 import { QTableProps } from 'quasar'
 import { defineComponent, ref, onMounted } from 'vue'
 
+interface Env {
+  [key: string]: string
+}
+
 export default defineComponent({
   name: 'IntEnvConfigComponent',
 
   setup() {
+    let selectedRows = ref([])
```

Comment:
You are correct. I think I had it in there because I was trying to use not `ref` objects and left it as a `let`. I never took a vue tutorial or anything, so maybe I should brush up on more of the underlying context behind some of these specifics 😢

Reply:
Yeah me too, I think there are probably some ref() I used elsewhere unnecessarily and will have to go back. Is great being able to work on this with others, raises my game. I looked at your code a while back on the SDK and it reminded me I am still far too reliant on promises .then .then .then. So went through and swapped a bunch out for async and await.